pkg/server/container_create_unix_test.go

```diff
@@ -367,7 +367,48 @@ func TestPodAnnotationPassthroughContainerSpec(t *testing.T) {
 			}
 		})
 	}
+}
+
+func TestContainerAnnotationPassthroughContainerSpec(t *testing.T) {
+	testID := "test-id"
+	testSandboxID := "sandbox-id"
+	testPid := uint32(1234)
 
+	for desc, test := range map[string]struct {
+		podAnnotations       []string
+		containerAnnotations []string
+		configChange         func(*runtime.PodSandboxConfig)
+		specCheck            func(*testing.T, *runtimespec.Spec)
+	}{
+		"passthrough annotations from pod and container should be passed as an OCI annotation": {
+			podAnnotations:       []string{"c"},
+			containerAnnotations: []string{"c*"}, // wildcard should pick up ca-c->ca-d pair in container
+			specCheck: func(t *testing.T, spec *runtimespec.Spec) {
+				assert.Equal(t, "d", spec.Annotations["c"])
```

Comment:
Can you add 2 test cases?
1) Annotations don't pass through if no pass through annotations are configured;
2) Unmatched annotations can't pass through even if pass through annotations are configured.

Reply:
done